### ISSUE TITLE
Cant reset query pool inside video coding block

### ISSUE THREAD

**Issue by friesofdoom** [nvpro-samples/vk_video_samples#67]
Reset happens here:
https://github.com/nvpro-samples/vk_video_samples/blob/9840c6416353d6b9359407711544e85af6527cfb/vk_video_encoder/libs/VkVideoEncoder/VkVideoEncoder.cpp#L1069

Begin of video coding block:
https://github.com/nvpro-samples/vk_video_samples/blob/9840c6416353d6b9359407711544e85af6527cfb/vk_video_encoder/libs/VkVideoEncoder/VkVideoEncoder.cpp#L959

**Comment by zlatinski** [comment]
Indeed, this is a bug per VUID-vkCmdResetQueryPool-videocoding. Fixed with 431de1d5ee54a77c1a1dddc15c320be7ad95f8cf.
Thank you, [friesofdoom](https://github.com/friesofdoom)!
